### ISSUE TITLE
Error downloading object

### ISSUE THREAD

**Issue by RealNee** [RedCommissary/altium-library#51]
Can not download footprint.PcbLib file.
Error downloading object: library/footprint.PcbLib (8a1fe72): Smudge error: Error downloading library/footprint.PcbLib (8a1fe720dfe0943e2a22110d08bc25987289f4078281e60659edd993b640ce66): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

**Comment by RedCommissary** [comment]
Yes, this problem with traffic LFS. Now it fixed and limit increase up to 50Gb. Thanks a lot!
